fastify.js

```diff
@@ -219,6 +219,8 @@ function fastify (options) {
     // custom parsers
     addContentTypeParser: ContentTypeParser.helpers.addContentTypeParser,
     hasContentTypeParser: ContentTypeParser.helpers.hasContentTypeParser,
+    defaultJSONParser: ContentTypeParser.defaultParsers.defaultJSONParser,
```

Comment:
This should be getDefaultJSONParser instead, and the two arguments should be passed over.

Reply:
@mcollina Which means the dev has to something like this ! fastify.getDefaultJSONParser (protoPoison, constructorPoison)(req, body, done). Is that Ok ?